Dockerfile

```diff
@@ -15,16 +15,27 @@ COPY docker/rootfs/ /
 RUN \
     apt-dpkg-wrap apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F23C5A6CF475977595C89F51BA6932366A755776 && \
     apt-dpkg-wrap apt-get update && \
-    apt-dpkg-wrap apt-get install -y build-essential python3.11 python3.11-venv && \
+    apt-dpkg-wrap apt-get install -y git wget build-essential git python3.11 python3.11-venv && \
```

Comment:
git is there twice.

Reply:
Also, don't we need the curl library, so the model tester works?

Reply:
actually that was three times, there was another git installed above :).

Wdym about the model tester?

Reply:
Don't we need libcurl so the demo page works?